packages/create-pantheon-decoupled-kit/src/templates/next-wp/.env.example

```diff
@@ -1,7 +1,21 @@
-# Optional environment variables are commented out
-WPGRAPHQL_URL=
-IMAGE_DOMAIN=
-#PANTHEON_UPLOAD_PATH=/example
-#PREVIEW_SECRET=
-#WP_APPLICATION_USERNAME=
-#WP_APPLICATION_PASSWORD=
+# Copy as .env.development.local to override envars for local development
+
+# The graphql endpoint of your WordPress instance
+WPGRAPHQL_URL=https://my-backend-site.pantheonsite.io/wp/graphql
+
+# The secret set on your WordPress Preview Site. Required for preview functionality
+PREVIEW_SECRET=super-secret-preview-secret
+
+# The domain where your images are hosted. By default, we will set the domain of the WPGRAPHQL_URL
+# to the value of IMAGE_DOMAIN. Set this if your images are hosted somewhere other than your WordPress instance
+# See https://nextjs.org/docs/pages/api-reference/components/image#domains for more information
+IMAGE_DOMAIN=some-other-domain.example
+
+# Username and Application Password for a valid user in the WordPress instance
+WP_APPLICATION_USERNAME=user
+WP_APPLICATION_PASSWORD=example_pass
+
+# The value of Next.js basePath.
+# See https://nextjs.org/docs/pages/api-reference/next-config-js/basePath for more information
+# This variable is only required if using the basePath feature
+PANTHEON_UPLOAD_PATH=/example
```

Comment:
How about 

> If you are using Next.js' `basePath` setting, then also set `PANTHEON_UPLOAD_PATH` to the same value. Setting `PANTHEON_UPLOAD_PATH` will [ok, I don't know what it'll do, but we should say what it does]. If you are not setting Next.js' `basePath`  then you probably don't need to set `PANTHEON_UPLOAD_PATH`.

Reply:
This var actually sets the `basePath`/`pathPrefix` for the user. how about:

```
# Sets the next.config.basePath setting for you
# This variable is only required if using the basePath feature
# See https://nextjs.org/docs/pages/api-reference/next-config-js/basePath for more information
```